server/service/apple_mdm.go

```diff
@@ -892,15 +893,7 @@ func (r enqueueMDMAppleCommandResponse) Status() int  { return r.status }
 
 func enqueueMDMAppleCommandEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (errorer, error) {
 	req := request.(*enqueueMDMAppleCommandRequest)
-	rawCommand, err := base64.RawStdEncoding.DecodeString(req.Command)
-	if err != nil {
-		return enqueueMDMAppleCommandResponse{Err: err}, nil
-	}
-	command, err := mdm.DecodeCommand(rawCommand)
-	if err != nil {
```

Comment:
Those validations resulted in `Forbidden` 403 responses because they happen before any authorization, so I refactored that a bit so that all the logic is in the service, and nothing happens before authorization (as this could be used by attackers e.g. to send big base64 data that we decode for nothing).